pkg/tree/schema.ipldsch

```diff
@@ -0,0 +1,41 @@
+type ProllyNode struct {
+    # true for leaf nodes and false for branch nodes
+    isLeaf Bool
+    # raw keys(input from users) for leaf node. For branch node, the key is last key in the child node
+	keys [Bytes]
+	# input data from users for leaf nodes, link(cid) for branch nodes
+    values [Any]
+} representation tuple
+
+type ProllyRoot struct{
+    # root ProllyNode of tree
+    RootCid &ProllyNode
+    # config about the tree
+    ConfigCid &TreeConfig
+}
+
+type SuffixThresholdConfig struct {
+    chunkingFactor Int
+} representation tuple
```

Comment:
Should also have hashFunction int https://github.com/ipld/ipld/pull/254/files#diff-b6100ac62d9896fb718b68b3208c91a1ba120b46253b7d95e0a65d4afaf72fe8R97